.github/workflows/integration.yml

```diff
@@ -170,6 +167,9 @@ jobs:
           REDSHIFT_TEST_USER: ${{ secrets.REDSHIFT_TEST_USER }}
           REDSHIFT_TEST_PORT: ${{ secrets.REDSHIFT_TEST_PORT }}
           REDSHIFT_TEST_HOST: ${{ secrets.REDSHIFT_TEST_HOST }}
+          DBT_TEST_USER_1: ${{ secrets.DBT_TEST_USER_1 }}
+          DBT_TEST_USER_2: ${{ secrets.DBT_TEST_USER_2 }}
+          DBT_TEST_USER_3: ${{ secrets.DBT_TEST_USER_3 }}
```

Comment:
We might not actually need these as repo secrets, versus just in plain text here

Reply:
Done